### PR TITLE
badge?

### PR DESCRIPTION
**Pull request by alexkar598**
# General Documentation

### Intent of your Pull Request

badge.

### Why is this change good for the game?

badge!

# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 
This will be the basis of the Wiki entry for your PR, and more information / detail is better for Wiki editors to integrate.

badge.

### What should players be aware of when it comes to the changes your PR is implementing?

¯\\\_(ツ)\_/¯

### What general grouping does this PR fall under? 
For example, Engineering changes, Medical rebalancing, TEG tweaks, etc.?

badge department

### Are there any aspects of the PR that you would like us not to mention on the Wiki?

the badges

### If there are any numerical values involved in your PR that will be relevant to a player, please note them here. 
For example, "This new chem will heal X brute damage". 

badges

# Changelog

:cl:  Badge Man
rscadd: badges
/:cl:

### CODE REVIEW

**Comment by alexkar598** [comment]
badge.

**Comment by Xoxeyos** [comment]
[**Badge.**](https://imgur.com/a/9dLSueI)

**Review by JamieD1 — APPROVED** [review]
